test/Makefile

```diff
@@ -15,54 +17,31 @@ endif
 ifndef LINUXKIT
 	$(error "linuxkit binary not found. please install it.")
 endif
+ifndef RTF
+	$(error "rtf is not available. please install it!")
+endif
+
 
+# TODO: Remove this section once we no longer depend on this in CI
+### -------
 define check_test_log
 	@cat $1 |grep -q 'test suite PASSED'
 endef
 
-../artifacts:
-	mkdir -p $@
-
-test-kernel-config-initrd.img: $(MOBY) cases/test-kernel-config.yml
-	$(MOBY) build --pull cases/test-kernel-config.yml
-
-# uses qemu
-.PHONY: kernel-config
-kernel-config: $(LINUXKIT) test-kernel-config-initrd.img test-kernel-config-kernel test-kernel-config-cmdline
-	$(LINUXKIT) run qemu test-kernel-config 2>&1 | tee test.log
-	$(call check_test_log, test.log)
-
-# qemu only at this time
-.PHONY: qemu-efi
-efi: $(LINUXKIT) test-kernel-config-efi.iso
-	$(LINUXKIT) run qemu -uefi test-kernel-config | tee test-efi.log
-	$(call check_test_log, test-efi.log)
-
-# For GCP tests
-../artifacts/test.img.tar.gz: test-kernel-config.img.tar.gz | ../artifacts
-	mv test-kernel-config.img.tar.gz ../artifacts/test.img.tar.gz
-
-.PHONY: gcp
-gcp: export CLOUDSDK_IMAGE_NAME?=test-kernel-config
-gcp: $(LINUXKIT) ../test.img.tar.gz
-	$(LINUXKIT) push gcp ../artifacts/test.img.tar.gz
-	$(LINUXKIT) run gcp $(CLOUDSDK_IMAGE_NAME) | tee test-gcp.log
-	$(call check_test_log, test-gcp.log)
-
-# For LTP tests
-test-ltp.img.tar.gz: $(MOBY) cases/test-ltp.yml
-	$(MOBY) build --pull cases/test-ltp.yml
-
-../artifacts/test-ltp.img.tar.gz: test-ltp.img.tar.gz | ../artifacts
-	mv test-ltp.img.tar.gz ../artifacts
-
 .PHONY: ltp
 ltp: export CLOUDSDK_IMAGE_NAME?=test-ltp
 ltp: $(LINUXKIT) test-ltp.img.tar.gz
```

Comment:
Yes, I have a feeling it's being used by CI to run the LTP tests so I didn't want to remove it just yet...